VERSION

```diff
@@ -1 +1 @@
-0.0.2
+0.0.3
```

Comment:
I don't know how this happened, but I think that once we are ready to release the update, we should increment this to 0.1.0